### ISSUE TITLE
Flags are not visible in Safari browser

### ISSUE THREAD

**Issue by pedrogarciyalopez** [AstrOOnauta/react-native-international-phone-number#43]
The TwemojiMozilla font does not work in the Safari browser. Check out your [demo](https://snack.expo.dev/@astroonauta/react-native-international-phone-number).
<img width="317" alt="image" src="https://github.com/AstrOOnauta/react-native-international-phone-number/assets/14232660/48bd8284-efe7-4a50-9769-9fa449a588d9">

**Comment by AstrOOnauta** [comment]
Hey @pedrogarciyalopez I was verify your case in my mac (usying my snack example)...and here worked (prints on end).

My Mac: MacOS13 (Ventura) with Safari v16.1.

🚀 AstrOOnauta 🚀

![image](https://github.com/AstrOOnauta/react-native-international-phone-number/assets/81580470/4293f867-d36c-45de-be2e-8335874463fd)
![image](https://github.com/AstrOOnauta/react-native-international-phone-number/assets/81580470/48384c25-2498-4260-ba5a-9b60b197e83f)

**Comment by AstrOOnauta** [comment]
@pedrogarciyalopez this is so insane haha...thanks for your investigation! We found a guilty...

I will search about this..if there are any solutions.
If you have any ideia how fix it...please, share with us =)

🚀 AstrOOnauta 🚀

**Comment by pedrogarciyalopez** [comment]
@AstrOOnauta, i think the quick solution is to use TwemojiMozilla for Windows only. Only there is it really necessary. Other platforms support emoji (flags) natively. Check out this [snack](https://snack.expo.dev/S26m6n7DL). Downside is inconsistent look across platforms.
<img width="990" alt="image" src="https://github.com/AstrOOnauta/react-native-international-phone-number/assets/14232660/ff688cf6-4439-49b7-8dc6-c1f318233cdb">

**Comment by AstrOOnauta** [comment]
@pedrogarciyalopez yes, this is a nice idea...but with RN I cannot detect if it is `web windows` or `web macOS`. Only `web`, usying the `Platform.OS` native method...

**Comment by pedrogarciyalopez** [comment]
@AstrOOnauta you can do something like this
```
const isWindowsWeb = Platform.OS === 'web' && navigator.userAgent.includes('Win');
```

**Comment by AstrOOnauta** [comment]
@pedrogarciyalopez Nice! Thanks for this insight...I will do this and return with any news =)

**Comment by pedrogarciyalopez** [comment]
@AstrOOnauta Great, thanks a lot!
